manual/manual/cmds/top.etex

```diff
@@ -246,6 +246,9 @@ Print version string and exit.
 \item["-vnum"]
 Print short version number and exit.
 
+\item["-no-version"]
+Do not print version string at startup.
```

Comment:
Why is this description different from the one in `ocaml.m`?
IMO they should both say something like "Do not print the version banner at startup."